participants/sanjitcodes.md

```diff
@@ -1,5 +1,26 @@
-Name: SANJIT ANAND
-GitHub username: sanjitcodes
-University: NIT, Surat
-Year: 1st
-Branch: Electronics and Communication
\ No newline at end of file
+# Introduction 
+##Name: SANJIT ANAND
+
+##GitHub username: sanjitcodes
+
+##University: NIT, Surat
+
+##Year: 1st
+
+##Branch: Electronics and Communication Engineering
+
+### I am from Amritsar, Punjab.
```

Comment:
@sanjitcodes nicely done! Just one more change and you are good to go. You need to add space between hashes and name, GitHub name, university, year and branch for markdown to consider it as H2 syntax